standalone/standalone.go

```diff
@@ -118,6 +122,24 @@ func Handler(ch grpcdynamic.Channel, target string, methods []*desc.MethodDescri
 	})
 }
 
+func registerExamplesHandler(mux *http.ServeMux, uiOpts *handlerOptions) {
+	var examplesBlob []byte
+	if uiOpts.examples != nil && len(*uiOpts.examples) != 0 {
+		marshaled, err := json.Marshal(uiOpts.examples)
+		if err == nil {
+			examplesBlob = marshaled
+		} else {
+			log.Printf("Failed to marshal examples: %v\n", err)
```

Comment:
This is not appropriate. You could instead save the error and use that to write an error code in response to `/examples` requests.